src/huntsman/pocs/tests/test_states.py

```diff
@@ -88,6 +101,26 @@ def test_parking_ready(pocs):
         assert pocs.state == state
 
 
+def test_parking_dome_status(pocs):
+    '''
+    Test if the dome is closed after parking state.
+    '''
+    pocs.initialize()
```

Comment:
It would be good to make sure the dome is actually open to begin with here.

e.g. `pocs.observatory.dome.open()` and/or `assert pocs.observatory.dome.is_open`

Reply:
The command `pocs.get_ready()` in the next line will open the dome. If it does not, the next test will raise an error!

Reply:
To prevent repetition in tests, I merge the dome `open` and `close` states tests in one test.